core/src/main/java/io/github/fourlastor/game/demo/state/unit/Mon.kt

```diff
@@ -38,12 +39,24 @@ class Mon(
 
     fun onRoundStart(): Action = stacks.onRoundStart(this)
 
+    fun onTurnStart(): Action = stacks.onTurnStart(this)
+
     fun onRoundEnd() {
         stacks.tickStacks()
     }
 
     fun addEffect(effect: Effect) {
-        stacks.addStack(effect, 3)
+        addEffect(effect, 3)
+    }
```

Comment:
I think we can inline this at this point (ctrl+shift+n with the cursor on the function name) and have only the one with the explicit parameter